test/conformance/ingress/emptyport.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ingress
+
+import (
+	"context"
+	"net"
+	"testing"
+	"time"
+
+	"google.golang.org/grpc"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"knative.dev/networking/pkg/apis/networking/v1alpha1"
+	"knative.dev/networking/test"
+	ping "knative.dev/networking/test/test_images/grpc-ping/proto"
+	"knative.dev/pkg/test/logstream"
+)
+
+// TestHTTP1AndEmptyPort verifies that an empty port name uses HTTP1. This should be the current behavior.
+func TestHTTP1AndEmptyPort(t *testing.T) {
```

Comment:
Can we add `t.Parallel()`?

Reply:
Will do! Sorry, I created these tests before that change went in, and missed copying it.